tests/support/Pipelines/Bbh/Test_InitialData.py

```diff
@@ -141,7 +141,7 @@ def test_cli(self):
             self.assertEqual(e.code, 0)
         with open(
             self.test_dir
-            / "Pipeline/001_InitialData/ControlParams_000/InitialData.yaml",
+            / "Pipeline/000_InitialData/ControlParams_000/InitialData.yaml",
```

Comment:
The remaining failure in this test is the assert in line 148 below: you have to add the new `"eccentricity_control": True` field.